### PR TITLE
Fixing comments

### PR DESCRIPTION
**Pull request by HannaMichaelis**
Removing all comments that mention C# 6.0 and earlier

Fixes Intellitect/EssentialCSharpManuscript#57

### CODE REVIEW

File: src/Chapter12/Listing12.02.SupportingUndoInEtchASketchGame.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public static void Sketch()
                         break;
 
                     default:
-                        Console.Beep();  // Added in C# 2.0
+                        Console.Beep();
```
Comment (BenjaminMichaelis) [inline src/Chapter12/Listing12.02.SupportingUndoInEtchASketchGame.cs]:
```suggestion
                        Console.Beep();
```
Nit: whitespace

File: src/Chapter12/Listing12.06.ImplementingUndoWithAGenericStackClass.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public static void Sketch()
             Stack<Cell> path = new Stack<Cell>();
             #endregion HIGHLIGHT
             Cell currentPosition;
-            ConsoleKeyInfo key;  // Added in C# 2.0
+            ConsoleKeyInfo key;
```
Comment (BenjaminMichaelis) [inline src/Chapter12/Listing12.06.ImplementingUndoWithAGenericStackClass.cs]:
```suggestion
            ConsoleKeyInfo key;
```
Nit: whitespace

File: src/Chapter12/Listing12.06.ImplementingUndoWithAGenericStackClass.cs [hunk 3]
```diff
@@ -105,7 +105,7 @@ public static void Sketch()
                         break;
 
                     default:
-                        Console.Beep();  // Added in C# 2.0
+                        Console.Beep();
```
Comment (BenjaminMichaelis) [inline src/Chapter12/Listing12.06.ImplementingUndoWithAGenericStackClass.cs]:
```suggestion
                        Console.Beep();
```
Nit: whitespace

**Review by BenjaminMichaelis — APPROVED** [review]

**Review by Keboo — APPROVED** [review]

**Comment by HannaMichaelis** [comment]
I'm confused, did this get merged or not?

**Comment by Keboo** [comment]
@HannaMichaelis this has not been merged yet.

**Comment by HannaMichaelis** [comment]
Ok, I don't know why that test isn't passing

**Pull request merged**